mentors/dart/j.json

```
@@ -15,4 +15,12 @@
     "avatar_url": "https://pbs.twimg.com/profile_images/791823568812187648/QM6VKGDq_400x400.jpg",
     "bio": "I've written a couple of blogs about Dart, and I've also contributed to the Dart track on Exercism!"
   }
```

```suggestion
  },
```
Just missing a comma here.